style.css

```diff
@@ -1,19 +1,84 @@
+:root {
+	--primary-background-color: hsl(24 20% 95%);
+	--primary-foreground-color: hsl(24 20% 5%);
+	--secondary-background-color: hsl(200 10% 20%);
+	--secondary-foreground-color: white;
+	--tertiary-background-color: transparent;
+	--tertiary-foreground-color: hsl(200 10% 20%);
+	--primary-shadow-color: white;
+	--secondary-shadow-color: hsl(0 0% 0% / .5);
+	--primary-border-color: hsl(0 0% 0% / .25);
+	--secondary-border-color: var(--secondary-background-color);
+	--shade-color: hsl(0 0% 0% / .3);
+
+	--spec-link-background-color: var(--secondary-background-color);
+	--spec-link-foreground-color: var(--secondary-foreground-color);
+	--highlight-background-color: #f06;
+	--highlight-foreground-color: white;
+	--prefix-background-color: var(--shade-color);
+	--prefix-foreground-color: var(--spec-link-foreground-color);
+	--progress-color: var(--shade-color);
+
+	--pass-color: #490;
+	--almost-pass-color: #8c0;
+	--slightly-buggy-color: #bb0;
+	--buggy-color: #e80;
+	--very-buggy-color: #e40;
+	--fail-color: #e20;
+	--epic-fail-color: #b00;
+
+	--aside-border-color: hsl(0 0% 80% / .8);
+	--aside-background-color: hsl(0 0% 98% / .6);
+	--aside-foreground-color: black;
+}
+
+@media (prefers-color-scheme: dark) {
+	:root {
+		--primary-background-color: hsl(24 20% 5%);
+		--primary-foreground-color: hsl(24 20% 95%);
+		--secondary-background-color: hsl(200 10% 20%);
+		--secondary-foreground-color: white;
+		--tertiary-background-color: transparent;
+		--tertiary-foreground-color: hsl(200 10% 90%);
+		--primary-shadow-color: black;
+		--secondary-shadow-color: hsl(0 0% 25% / .5);
+		--primary-border-color: hsl(0 0% 25% / .25);
+		--secondary-border-color: var(--secondary-background-color);
+
+		--pass-color: #370;
```

Comment:
The hex kinda masks what's going on here. Do these only differ in lightness, or also other components?

Reply:
I used hex colors here because that's what you used for those colors.

The idea was to make them a little darker but I didn't really stick to a specific schema here. I might switch to `hsl()` here and dim the luminance a bit if you think that's better.

Reply:
I do think HSL will make it clearer what's going on, yes.

Reply:
Done.